.yarnrc.yml

```diff
@@ -1,19 +1,7 @@
-nmHoistingLimits: workspaces
```

Comment:
This was a workaround for a Storybook issue that has since been resolved. Removing this setting causes most packages to hoist unless there's a specific version difference. This also solved the plugin-prettier-svelte issue with lerna.